plugin/kubernetes/handler_test.go

```diff
@@ -251,7 +251,9 @@ var dnsTestCases = []kubeTestCase{
 	{Case: test.Case{
 		Qname: "kubernetes.default.svc.cluster.local.", Qtype: dns.TypeTXT,
 		Rcode:  dns.RcodeSuccess,
-		Answer: []dns.RR{},
+		Answer: []dns.RR{
+			test.SOA("cluster.local.	5	IN	SOA	ns.dns.cluster.local. hostmaster.cluster.local. 1499347823 7200 1800 86400 5"),
+        },
```

Comment:
This doesn't look right.

For a NODATA response, the SOA record should be in the Authority section (`Ns`).